frappe/public/js/frappe/form/formatters.js

```diff
@@ -103,8 +103,9 @@ frappe.form.formatters = {
 	},
 	Currency: function (value, docfield, options, doc) {
 		var currency = frappe.meta.get_field_currency(docfield, doc);
-		var precision =
-			docfield.precision || cint(frappe.boot.sysdefaults.currency_precision) || 2;
+		var precision = cint(
+			docfield.precision ?? frappe.boot.sysdefaults.currency_precision ?? 2
```

Comment:
I've made changes to simplify the code. `??` does what you want to achieve without complicating it. 


https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
I have tested this code locally, and it perfectly and succinctly solves my needs. Thank you for your help 👍